release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -420,13 +422,27 @@ Red Hat does not guarantee backward compatibility for recording rules or alertin
 ** The `NodeRAIDDegraded` and `NodeRAIDDiskFailure` alerts now include a device label filter to match only the value returned by `mmcblk.p.+|nvme.+|sd.+|vd.+|xvd.+|dm-.+|dasd.+`.
 ** The `PrometheusHighQueryLoad` and `ThanosQueryOverload` alerts now also trigger when a high querying load exists on the query layer.
 
+[id="ocp-4-12-monitoring-new-option-to-specify-pod-topology-spread-constraints-for-monitoring-components"]
 ==== New option to specify pod topology spread constraints for monitoring components
 You can now use pod topology spread constraints to control how Prometheus, Thanos Ruler, and Alertmanager pods are spread across a network topology when {product-title} pods are deployed in multiple availability zones.
 
+[id="ocp-4-12-monitoring-new-option-to-improve-data-consistency-for-prometheus-adapter"]
 ==== New option to improve data consistency for Prometheus Adapter
 You can now configure an optional kubelet service monitor for Prometheus Adapter (PA) that improves data consistency across multiple autoscaling requests.
 Enabling this service monitor eliminates the possibility that two queries sent at the same time to PA might yield different results because the underlying PromQL queries executed by PA might be on different Prometheus servers.
 
+[id="ocp-4-12-monitoring-update-to-alertmanager-template-configuration-for-additional-secret-keys"]
+==== Update to Alertmanager template configuration for additional secret keys
+With this release, if you configure an Alertmanager secret to hold additional keys and if the Alertmanager configuration references these keys as templates, your configuration settings must point to these keys by using an absolute path rather than a relative path.
```

Comment:
In theory, users could use additional data for more than just templates.
```suggestion
With this release, if you configure an Alertmanager secret to hold additional keys and if the Alertmanager configuration references these keys as files (such as templates, TLS certificates, tokens), your configuration settings must point to these keys by using an absolute path rather than a relative path.
```